core/src/main/java/com/linecorp/armeria/common/metric/MoreMetrics.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2023 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.common.metric;
+
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.netty.channel.EventLoopGroup;
+
+/**
+ *  Provides useful {@link io.micrometer.core.instrument.binder.MeterBinder}s to monitor
+ *  various Armeria components.
+ */
+public final class MoreMetrics {
+
+    /**
+     * A {@link MeterBinder} to observe Netty's {@link EventLoopGroup}s. The following stats are currently
+     * exported per registered {@link MeterIdPrefix}.
+     *
+     * <p>
+     *  - "event.loops.num.workers" (gauge) - the total number of Netty's event loops
+     *  - "event.loops.pending.tasks" (gauge) - the total number of IO tasks waiting to be run on event loops*
+     * </p>
+     */
+    public static MeterBinder eventLoopMetrics(MeterIdPrefix idPrefix, EventLoopGroup eventLoopGroup) {
```

Comment:
What do you think about merging this class into `MoreMeters`?

Reply:
This would be the first method there returning MeterBinder (or metrics in that sense).

MoreMeters seems to mostly contain utilities to work with `MeterRegistry`, which is also suggested by its javadoc:

```
// Provides utilities for accessing MeterRegistry.
```

I'd probably vote for keeping `MoreMetrics` as a factory for creating MeterBinders.

Reply:
IMHO, it's better to merge those classes because it can be a bit distracting for users to find Micrometer relates utilities. We can fix the Javadoc of `MoreMeters` if that's the problem. 😄 
Anyway, we still need to add `@UnstableApi` because this is a newly-added public method. 😉

Reply:
Added `@UnstableApi` annotation for `eventLoopMetrics`.